src/app/domain/schema-management/components/manual-verification-wizard.component.html

```
@@ -0,0 +1,122 @@
+            <thead class="bg-gray-50 dark:bg-slate-900/50">
+              <tr>
+                <th scope="col" class="px-4 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">Verify</th>
+                <th scope="col" class="px-4 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">Row</th>
+                <th scope="col" class="px-4 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">Subject ID</th>
+                <th scope="col" class="px-4 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">Treatment</th>
+                <th scope="col" class="px-4 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">Strata</th>
+              </tr>
+            </thead>
+            <tbody class="bg-white dark:bg-slate-800 divide-y divide-border-subtle">
+              @for (sample of samples(); track sample.rowIndex; let i = $index) {
+                <tr [class.bg-green-50]="sample.verified" [class.dark:bg-green-900/20]="sample.verified" class="transition-colors">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)